SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -147,11 +147,6 @@ Interest InterestFactory::CreateServerWorkerInterest()
 
 Interest InterestFactory::CreateInterest() const
 {
-	if (!GetDefault<USpatialGDKSettings>()->bUsingQBI)
-	{
-		return Interest{};
-	}
-
 	if (GetDefault<USpatialGDKSettings>()->bEnableServerQBI)
```

Comment:
What happens when `bEnableServerQBI` is false? Does it just check out everything?

Reply:
Yep (each entity gets an interest query with component constraint for the spatial position)

Reply:
Oh, I think you mean the special server entity has an interest query with a component constraint of spatial position. I think I get what's going on there now.

Reply:
That is indeed what I meant. (With server QBI on, each entity gets an interest query)